### DISCUSSION TITLE
Suggestions category

### DISCUSSION THREAD

**Discussion by tijno** [deso-protocol/dips#79] [Q&A] [answered]
The CIPS discussion category is getting filled up quit nicely, but sometimes they are just suggestions or feature requests with little implementation details.

Would it be worth adding a "Suggest" category to the Discussions @maebeam and move some of the feature requests there - so CIPs is more focussed on those proposals with sufficient details?

**Comment by maebeam** [comment, marked as answer]
Suggestions feel like medium level CIPs which should be separated from large CIPs. Should we have a Medium and Large category?

**Reply by tijno** [reply]
yes that would work

**Reply by bloated-devish** [reply]
my 2 cents:
I think CIPs should have 2 levels:
1) Product proposal (similar to PRD).
2) Eng design.

What we are seeing is more of 1 and less of 2. 1 is generally enough for small to medium features, but for large ones, we will need both 1,2. Without this, it would be hard to implement large features. 

My thought is that we should still encourage people to do 1, as this would be helpful for everyone to contribute, even if they don't have strong tech skills. May be we should add a tag as - need eng design - so devs who are eager to contribute can see through this proposal and add the dev design if the interest warms up, so at the end, we have both 1,2 in all good ideas.

**Reply by maebeam** [reply]
I created a new Large CIPs category. Let's see how this goes

**Comment by omicreativedev** [comment]
What about...
Q&A (without template)
Suggestion (basic template - problem/solution)
CIP (with full template)

Then
Small tag - any approved can be picked up by anyone in community
Medium tag - has to have 1 core approval 
Large tag - has to have 2 core approval + # of community

**Reply by tijno** [reply]
this would be a good approach too

tags especially

note that:

small dont need discussion and can go straight to submitting PR in relevant repo with changes.

CIP template was intended i think for PR to CIP repo after Discussion has reached required approvals
